src/main/java/batman/exception/Error.java

```diff
@@ -0,0 +1,7 @@
+package batman.exception;
+
+public enum Error {
+    INVALID,
+    EMPTYDESC,
+    LISTERROR
```

Comment:
Shouldn't your constants name be all uppercase using underscore to separate words? -> EMPTY_DESC and LIST_ERROR